### DISCUSSION TITLE
[Project Beta] Add color to labels

### DISCUSSION THREAD

**Discussion by allenchuang** [community/community#13376] [Projects and Issues]
Like the title, we should be able to set color for diff labels, its all monotone rn and working on projects should be fun! 🚀

**Comment by vvdavidova** [comment]
@allenchuang, you should be able to add colours to different labels. 
Go to your organisation, click repositories, select the repository in which you want to edit the labels, click issues, then click labels. 
You should be able to add new/edit existing labels and one of the features for both options is changing the colour. 
I hope that helps.
